### ISSUE TITLE
WPF offline map

### ISSUE THREAD

**Issue by Kayumiy** [fatihgvn/cSharp-offline-Gmap#1]
I applied your approach to my project which is WPF project.
It does not work.

**Comment by fatihgvn** [comment]
Yeah.  because libraries are written for form application.  I haven't found the solution yet.  I couldn't care enough.
